libs/application-generic/src/utils/index.ts

```diff
@@ -12,3 +12,4 @@ export * from './subscriber';
 export * from './variants';
 export * from './deepmerge';
 export * from './generate-id';
+export * from './sanitize-preview-control-values';
```

Comment:
I suggest renaming this file to `sanitize-control-values.ts`. If not mistaken, it should be reused for preview and actual execution. A more generic name will work better.